src/main/java/me/zeroeightsix/kami/gui/mc/KamiGuiAntiDisconnect.kt

```diff
@@ -48,6 +42,8 @@ class KamiGuiAntiDisconnect : GuiScreen() {
                             mc.displayGuiScreen(GuiMultiplayer(GuiMainMenu()))
                         }
                     }
+                    mc.world.sendQuittingDisconnectingPacket()
```

Comment:
why are you sending these after?

Reply:
I think that otherwise it believes that you have already left, therefore no integrated server is running, at least this worked in my test and it didn't with these before.

Reply:
I think that was the intention. I send it in the old order for autolog in order to log out of the server as quickly as possible, I didn't find any issue with doing it the old way

Reply:
I will test it further, however I believe that this was the reason it wasn't working.

Reply:
Wait, also why do you need anti disconnect to log out quickly? Isn't it meant for stopping that.

Reply:
I'm saying that AutoLog uses the same order (before you changed it) to log out as fast as possible, and it doesn't have problems.

Reply:
Just tested it, yes this does cause an issue having this bit before.